kaldb/src/main/proto/trace.proto

```diff
@@ -57,12 +57,12 @@ message Span {
   // For example: a value of 1551849569000000 represents March 6, 2019 5:19:29 UTC.
   // We use fixed64 since it is more wire efficient for than int64 for larger numbers.
   // This field is required.
-  fixed64 start_timestamp_micros = 5;
+  fixed64 timestamp = 5;
```

Comment:
https://github.com/openzipkin/zipkin-api/blob/master/zipkin.proto#L123-L129

this is what the name of the field is on the zipkin proto ( also linked in this proto file )